research/minigo/minigo.py

```diff
@@ -251,6 +231,11 @@ def validate(trained_models_dir, holdout_dir, estimator_model_dir, params):
         tf_records.extend(
             tf.gfile.Glob(os.path.join(record_dir, '*'+_TF_RECORD_SUFFIX)))
 
+  if len(tf_records) is 0:
```

Comment:
I think the quickest way is "if not tf_records".

Reply:
Yeah, thanks!